lib/ontologies_linked_data/models/mod/semantic_artefact.rb

```diff
@@ -116,6 +116,12 @@ def attribute_mapped(name, **options)
             attribute_mapped :logo, namespace: :foaf, mapped_to: {model: :ontology_submission, attribute: :logo}
             attribute_mapped :metrics, namespace: :mod, mapped_to: {model: :ontology_submission, attribute: :metrics}
 
+            attribute_mapped :numberOfNotes, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfNotes}
+            attribute_mapped :numberOfUsingProjects, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfUsingProjects}
+            attribute_mapped :numberOfEndorsements, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfEndorsements}
+            attribute_mapped :numberOfEvaluations, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfEvaluations}
+            attribute_mapped :numberOfUsers, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfUsers}
+            attribute_mapped :numberOfAgents, namespace: :mod, enforce: [:integer],  mapped_to: {model: :metric, attribute: :numberOfAgents}
```

Comment:
here you can save some text, by taking by default the name of the mapped attribute  as they are the same, to be like bellow
```ruby 
attribute_mapped :numberOfAgents, namespace: :mod, enforce: [:integer], mapped_to: :metric 
```
this shortcut can be applied only when the mapped attribute and the mapped_to attribute are the same.